vendor.conf

```diff
@@ -115,7 +115,7 @@ github.com/googleapis/gax-go v2.0.0
 google.golang.org/genproto 694d95ba50e67b2e363f3483057db5d4910c18f9
 
 # containerd
-github.com/containerd/containerd v1.2.0-beta.2
+github.com/containerd/containerd d97a907f7f781c0ab8340877d8e6b53cc7f1c2f6
```

Comment:
Is a cherry-pick opened for the patch to get it into the 1.2 branch in containerd?

Reply:
@dmcgowan Not sure how you're cutting 1.2, or just refreshing from master on each release?

Reply:
Oh, nevermind, I see that the 1.2-beta's are still tagged from master, and that there's no 1.2 branch yet